src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -98,20 +97,13 @@
     private static final Log log = Log.getInstance(CollectAlignmentSummaryMetrics.class);
 
     @Option(doc="Paired-end reads above this insert size will be considered chimeric along with inter-chromosomal pairs.")
-    public int MAX_INSERT_SIZE = 100000;
+    public int MAX_INSERT_SIZE = ChimeraUtil.DEFAULT_INSERT_SIZE_LIMIT;
 
     @Option(doc="Paired-end reads that do not have this expected orientation will be considered chimeric.")
-    public PairOrientation EXPECTED_PAIR_ORIENTATION = PairOrientation.FR;
+    public Set<PairOrientation> EXPECTED_PAIR_ORIENTATIONS = ChimeraUtil.DEFAULT_EXPECTED_ORIENTATIONS;
```

Comment:
To be safe, this should probably copy-construct the set of orientations.  I believe EnumSet is mutable, and that CommandLineProgram will mutate it!